source/howto/data/tag-files.rst

```diff
@@ -94,7 +94,8 @@ You can access the files metadata using Valohai `DatumList API <https://app.valo
     import json
 
     metadata = {
-        "valohai.tags": ["prod", "lemonade"], #creates Valohai tags for the file
+        "valohai.tags": ["prod", "lemonade"], # creates Valohai tags for the file
+        "valohai.alias": ["model-prod"], # creates or updates a Valohai alias to point to this file
```

Comment:
I'd change the comment to `# creates or updates a Valohai data alias to point to this output [file]` with [file] being optional 'file' or empty